developer-guides/rest-api/channels/counters/README.md

```diff
@@ -0,0 +1,56 @@
+# Channel Counters
+
+Gets channel counters.
+
+| URL | Requires Auth | HTTP Method |
+| :--- | :--- | :--- |
+| `/api/v1/channels.counters` | `yes` | `GET` |
+
+## Payload
+
+One of roomId or roomName is required.
+
+| Argument | Example | Required | Description |
+| :--- | :--- | :--- | :--- |
+| `roomId` | `GENERAL` | Required | The channel's id |
+| `roomName` | `general` | Required | The channel's name |
+| `userId` | `RtycPC29hqLJfT9xj` | Optional | Counters for provided user id (need to have a view-room-administration right for calling user) |
+
+## Example Call
+
+```bash
+curl -H "X-Auth-Token: 8h2mKAwxB3AQrFSjLVKMooJyjdCFaA7W45sWlHP8IzO" \
+     -H "X-User-Id: ew28FnZqipDpvKw3R" \
+     -H "Content-type: application/json" \
+     http://localhost:3000/api/v1/channels.counters?roomId=GENERAL
+```
+
+## Example Result
+
+```json
+{
+   "joined":true,
+   "members":59,
+   "unreads":0,
+   "msgs":236,
```

Comment:
Hi @xbolshe maybe you forgot `unreadsFrom` property in this response.

Reply:
Hi @MarcosSpessatto Thanks for a detection of issue. Now it is fixed as well as docs.